bisect.ini.sample

```diff
@@ -16,6 +16,7 @@
 #   "master @ October 2013", "2.065 @ 15 days ago" or "@ Tuesday")
 #   or a tag (e.g. "master @ v2.085.0).
 #   If no ref is specified, master is assumed.
```

Comment:
Maybe changing this line to say "If no starting point is specified, master is assumed" instead will help?

Reply:
I'll rather close. At least I understand better how it works.